### PR TITLE
Sprints/adulthood

### PR DESCRIPTION
**Pull request by pierluc-ssoucy**

### CODE REVIEW

File: src/Compliance.Solution/Solution/Other/Relationships.xml [hunk 1]
```diff
@@ -338,8 +341,7 @@
   <EntityRelationship Name="msdyn_aifptrainingdocument_Annotations" />
   <EntityRelationship Name="msdyn_aimodel_Annotations" />
   <EntityRelationship Name="msdyn_aiodimage_Annotations" />
-  <EntityRelationship Name="msdyn_solutioncomponentdatasource_Annotations" />
-  <EntityRelationship Name="msdyn_solutionhistorydatasource_Annotations" />
+  <EntityRelationship Name="msdyn_knowledgearticletemplate_SharePointDocumentLocations" />
```
Comment (pierluc-ssoucy) [inline src/Compliance.Solution/Solution/Other/Relationships.xml:344]: @mitchkirady i don't think this one is supposed to be added?

Comment (mitchkirady) [reply src/Compliance.Solution/Solution/Other/Relationships.xml:344]: that's correct, we should prevent unmodified OOB relationships from being added to our solution. Not sure why these are being added.

File: src/Ci2.Migration/Analytics/unreferenced_entity.sql [hunk 2]
```diff
@@ -21,9 +22,9 @@ DECLARE cMetadata CURSOR FOR
 			JOIN [Ci2_MSCRM].[MetadataSchema].[Entity] AS e2 ON r.ReferencedEntityId = e2.EntityId
 			JOIN [Ci2_MSCRM].[MetadataSchema].[Attribute] AS a ON r.ReferencingAttributeId = a.AttributeId AND a.TableColumnName IS NOT NULL
 			JOIN [Ci2_MSCRM].[MetadataSchema].[Attribute] AS a2 ON r.ReferencedAttributeId = a2.AttributeId AND a2.TableColumnName IS NOT NULL
-			WHERE r.ReferencedEntityId = @TargetEntityTypeId
+			WHERE (r.ReferencedEntityId = @TargetEntityTypeId or r.ReferencedEntityId = @TargetEntityTypeIdActivityParty)
 				AND e.BaseTableName NOT IN (
-				'ActivityPartyBase',
+				'ActivityParty',
```
Comment (EvoPheonix) [inline src/Ci2.Migration/Analytics/unreferenced_entity.sql:27]: @EvoPheonix should not have changed, should be 'ActivityPartyBase'

File: src/Compliance.EarlyBound/Entities/UserSettings.cs [hunk 3]
```diff
@@ -890,6 +890,26 @@ public System.Nullable<System.DateTime> LastAlertsViewedTime
 			}
 		}
 		
+		/// <summary>
+		/// Stores the timestamp for when the ViewPersonalizationSettings attribute was updated for this user in the UserEntityUISettings table.
+		/// </summary>
+		[Microsoft.Xrm.Sdk.AttributeLogicalNameAttribute("lastmodifiedtimeforviewpersonalizationsettings")]
+		public System.Nullable<System.DateTime> LastModifiedTimeForViewPersonalizationSettings
```
Comment (EvoPheonix) [inline src/Compliance.EarlyBound/Entities/UserSettings.cs:897]: this does not follow the naming convention

File: src/Compliance.Solution/Solution/Other/Relationships/opc_RiskAssessment.xml [hunk 4]
```diff
@@ -18,7 +18,7 @@
     <RelationshipDescription>
       <Descriptions>
         <Description description="Reference to the Risk Assessment." languagecode="1033" />
-        <Description description="Référence à l’évaluation des risques." languagecode="1036" />
+        <Description description="Renvoi à l’évaluation des risques." languagecode="1036" />
```
Comment (JonathanThomassin) [inline src/Compliance.Solution/Solution/Other/Relationships/opc_RiskAssessment.xml:21]: I dont know why this keep changing but it should stay Référence

File: src/Compliance.Solution/Solution/SdkMessageProcessingSteps/{724c4012-55ea-eb11-bacb-000d3a09c88d}.xml [hunk 5]
```diff
@@ -1,29 +0,0 @@
...
-    <SdkMessageProcessingStepImage Name="PreTarget">
-      <SdkMessageProcessingStepImageId>{76fb56ea-70fc-eb11-94ef-0022486dba15}</SdkMessageProcessingStepImageId>
-      <Attributes></Attributes>
-      <EntityAlias>PreTarget</EntityAlias>
-      <ImageType>0</ImageType>
-      <MessagePropertyName>Target</MessagePropertyName>
-      <IsCustomizable>1</IsCustomizable>
-    </SdkMessageProcessingStepImage>
-  </SdkMessageProcessingStepImages>
-</SdkMessageProcessingStep>
```
Comment (JonathanThomassin) [inline src/Compliance.Solution/Solution/SdkMessageProcessingSteps/{724c4012-55ea-eb11-bacb-000d3a09c88d}.xml:29]: We need to add this back.

File: src/Compliance.Solution/Solution/SdkMessageProcessingSteps/{5039484d-5e00-ec11-94f0-0022486d62b6}.xml [hunk 6]
```diff
@@ -1,20 +0,0 @@
...
-  <Mode>0</Mode>
-  <Rank>1</Rank>
-  <EventHandlerTypeCode>4602</EventHandlerTypeCode>
-  <Stage>40</Stage>
-  <IsCustomizable>1</IsCustomizable>
-  <IsHidden>0</IsHidden>
-  <SupportedDeployment>0</SupportedDeployment>
-  <IntroducedVersion>1.0</IntroducedVersion>
-  <SdkMessageProcessingStepImages />
-</SdkMessageProcessingStep>
```
Comment (JonathanThomassin) [inline src/Compliance.Solution/Solution/SdkMessageProcessingSteps/{5039484d-5e00-ec11-94f0-0022486d62b6}.xml:20]: We need to add this back.

File: src/Compliance.EarlyBound/Entities/UserSettings.cs (hunk 3 above)
Comment (pierluc-ssoucy) [reply src/Compliance.EarlyBound/Entities/UserSettings.cs:897]: this seems to be some sort of out of the box field

**Pull request merged**